core/src/main/java/com/datastrato/gravitino/catalog/FilesetStandardizedDispatcher.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog;
+
+import static com.datastrato.gravitino.catalog.CapabilityHelpers.applyCapabilities;
+
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.Namespace;
+import com.datastrato.gravitino.connector.capability.Capability;
+import com.datastrato.gravitino.exceptions.FilesetAlreadyExistsException;
+import com.datastrato.gravitino.exceptions.NoSuchFilesetException;
+import com.datastrato.gravitino.exceptions.NoSuchSchemaException;
+import com.datastrato.gravitino.file.Fileset;
+import com.datastrato.gravitino.file.FilesetChange;
+import java.util.Map;
+
+public class FilesetStandardizedDispatcher implements FilesetDispatcher {
```

Comment:
How about `FilesetNormalizeDispatcher`, change to `xxxNormalizeDispatcher`?

Reply:
updated